### PR TITLE
use message for updating loadflowbutton state

### PR DESCRIPTION
**Pull request by JacquesBorsenberger**
Signed-off-by: jacqbors <jacques.borsenberger@rte-france.com>

# Conflicts:
#	src/components/study-pane.js

### CODE REVIEW

File: src/components/study-pane.js [hunk 1]
```diff
@@ -99,7 +108,7 @@ const StudyPane = () => {
 
     const [filteredNominalVoltages, setFilteredNominalVoltages] = useState([]);
 
-    const [loadFlowRunning, setLoadFlowRunning] = useState(false);
+    const [loadFlowRunning, setLoadFlowRunning] = useState(LFStatus.NOT_RUN);
```
Comment (ChamseddineBhd) [inline src/components/study-pane.js:111]: loadFlowState ?

**Review by ChamseddineBhd — CHANGES_REQUESTED** [review]

File: src/components/study-pane.js [hunk 2]
```diff
@@ -71,6 +73,13 @@ const useStyles = makeStyles((theme) => ({
 const INITIAL_POSITION = [0, 0];
 
 const StudyPane = () => {
+    const LFStatus = {
+        CONVERGED: 'Converged',
+        DIVERGED: 'Diverged',
+        NOT_RUN: 'Start LoadFlow',
```
[the remark this answers is not in the document]
Comment (JacquesBorsenberger) [reply src/components/study-pane.js:79]: it's the initial state, when the loadflow has not been run, CONVERGED & DIVERGED are also 'not running'

File: src/components/study-pane.js (hunk 1 above)
Comment (JacquesBorsenberger) [reply src/components/study-pane.js]: done

**Review by ChamseddineBhd — APPROVED** [review]

**Review by mikatheo — CHANGES_REQUESTED** [review]
Really nice!
It works rather well, however I have 2 remarks:
- When the Loadflow is finished, the network image loses the voltage level selection (all are displayed)
- The colors grey / green / red / orange make it difficult to distinguish the label, could the contrast between the two be enhanced?

**Review by geofjamg — APPROVED** [review]

**Pull request merged**